golang/cosmos/x/vtransfer/keeper/keeper.go

```diff
@@ -200,27 +233,36 @@ func (k Keeper) InterceptWriteAcknowledgement(ctx sdk.Context, chanCap *capabili
 	return nil
 }
 
-// parseTransfer checks if a packet's sender and/or receiver are targeted accounts.
-func (k Keeper) parseTransfer(ctx sdk.Context, packet ibcexported.PacketI) (string, string, error) {
+// findTransferTargets checks if a packet's sender and/or receiver correspond to targeted accounts.
+func (k Keeper) findTransferTargets(ctx sdk.Context, packet ibcexported.PacketI) (string, string, error) {
```

Comment:
```suggestion
// findTransferTargets returns the targeted accounts (if any) corresponding with
// a packet's sender and/or receiver.
func (k Keeper) findTransferTargets(ctx sdk.Context, packet ibcexported.PacketI) (
	senderTarget string, receiverTarget string, err error) {
```
(with corresponding `:=` → `=` changes in the body)

Reply:
I simplified this worker function to just `isTargetWatched` and moved all the parsing logic into `types/baseaddr.go`.